xmodule/css/tabs/tabs.scss

```diff
@@ -130,10 +130,6 @@
     padding: 0;
     border: none;
   }
-
-  .blue-button {
-    @include blue-button;
-  }
```

Comment:
Sorry, I am having trouble understanding what you mean here:

> That was happening because the import order had changed and an specific button mixin was not overridden by bourbon, the file that generates that difference is [this one](https://github.com/openedx/edx-platform/blob/master/xmodule/css/tabs/tabs.scss#L135), that use the [blue-button](https://github.com/openedx/edx-platform/blob/master/common/static/sass/_mixins-inherited.scss#L191) mixin, and that mixin use inside the button mixin in this [line](https://github.com/openedx/edx-platform/blob/master/common/static/sass/_mixins-inherited.scss#L192), when the bourbon import is made in the file openedx-assets xmodule that override the button mixin with [this](https://github.com/openedx/edx-platform/blob/master/common/static/sass/bourbon/addons/_button.scss#L1) however when I applied this changes that uses this [mixin](https://github.com/openedx/edx-platform/blob/master/common/static/sass/_mixins-inherited.scss#L134)

As I understand it, you are removing this block because it generates extra CSS in the video editor (which is the only thing that uses tabs.scss), but that CSS has no visual impact on the video editor, right?

Reply:
Sounds good to me 👍🏻